File: Cpgrms/palindrome.c
#include <stdio.h>
void main()
{

    int n,rem,rev=0,temp;
    printf("enter a number \n");
    scanf("%d",&n);

    while(n!=0)
    {
        rem=n%10;
        rev=rev*10+rem;
        n=n/10;
    }
    if(n==rev)
    {
        printf("is palindrome");

    }
    else
    {
        printf("is not palindrome");
    }
}

File: pythonProject/CO1_CO2/palindrome.py
n=int(input("enter a number to reverse"))
rev=0
temp=n
while(n!=0):
    rem=n%10
    rev=rev*10+rem
    n//=10
if(rev==temp):
    print("number is palindrome",rev)

else:
    print("number is not a palindrome",rev)

